src/components/views/elements/TextWithTooltip.tsx

```diff
@@ -28,19 +29,18 @@ interface IProps extends HTMLAttributes<HTMLSpanElement> {
 }
 
 export default class TextWithTooltip extends React.Component<IProps> {
+    private id = "mx_TextWithTooltip_" + randomString(8);
```

Comment:
Is there a reason why TooltipTarget doesn't automatically generate an ID internally?

Reply:
Good point